frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -114,15 +114,26 @@ export const BundleBanner = (props: Props) => {
                 </Localized>
               </h2>
               <h3>{l10n.getString("bundle-banner-subheader")}</h3>
-              <p>{l10n.getString("bundle-banner-body")}</p>
-              <Localized
-                id={"bundle-banner-1-year-plan-v2"}
-                elems={{
-                  b: <b />,
-                }}
-              >
-                <p />
-              </Localized>
+              <p>
+                {l10n.getString("bundle-banner-body-v2", { savings: "40%" })}
+              </p>
+              <p>
+                <strong>{l10n.getString("bundle-banner-plan-header")}</strong>
+              </p>
+              <div className={styles["bundle-banner-value-props"]}>
+                <div>
```

Comment:
I feel like this would make more sense as a `<ul>` with `<li>`s in there, given that it's a list of included products? (With of course the same change applied to the CSS selectors.) Right now I think it reads a bit weird for screen reader users.

Reply:
I was thinking about that using unordered list that but wasn't sure if it was the right use case. Good callout.